uiowa/uiowa
# Remove field_uiowa_events_past

Resolves #5518 

Removes the deprecated `field_uiowa_events_past` boolean field, and its related logic in `layout_builder_custom`.





# How to test



- `git checkout remove-events-past-bool && git pull`
- Sync a site, watch that the config update works and completes successfully
- Test an existing events block and create a new one, check that they still work
- Sync a v2 site, double check that the events paragraph still works
- Double check that the db no longer contains data for the old field
  - Data would normally be stored in `block_content_revision__field_uiowa_events_past` and `block_content__field_uiowa_events_past`, both of which should be removed upon removal of the field from config
- Double check for other custom code that might be tied to this field (`field_uiowa_events_past`)
